_layouts/project.html.haml

```diff
@@ -104,7 +104,7 @@
                 %li
                   %a(href="https://issues.jboss.org/browse/JBTM") Issue Tracker
                 %li{:class => page.output_path.start_with?("/lra") ? "active": ""}
-                  %a{:href => relative("/lra/certifications/index.html")} Certifications
+                  %a{:href => relative("/lra/index.html")} LRA
```

Comment:
@ochaloup - I wonder if we can keep a certifications link on this top bar and try to put the general LRA content under https://github.com/jbosstm/narayana.io/blob/develop/sub-projects/index.html.haml ?

Reply:
hi @tomjenkinson 

I did following

* moved the `/lra/certifications` folder under just `/certifications`
* moved the content of openapi yamls to `lra/1.0`
* created a "Long Running Label" label under the `Documentation` subpage wher I added the link to right column under `Other Resources`
* made active the `Documentation` top menu bar navigation when working with Documentation "subpages"

Reply:
please don't do this as this is already linked in the CCR. We shouldn't change the certifications paths on fly as old links should continue working even years after the specification is released. I can still change it now but not later than the ballot is started.

Reply:
Am I right in thinking all we need to do is make sure is that https://narayana.io/lra/certifications/microprofile-lra-1.0/index.html remains valid?

Reply:
Assuming that is right, can't we just remove the move of the folder `lra/certifications/microprofile-lra-1.0` to `certifications/microprofile-lra-1.0` (and any link changes) from this PR?

Reply:
@tomjenkinson yes you care correct. It doesn't necessarily need to be this one yet as the ballot still hasn't started (issues with EF) so we have unknown time now to fix it so I can update it in CCR. But after that it shouldn't ever change.

Reply:
I don't have a problem with https://narayana.io/lra/certifications/microprofile-lra-1.0/index.html specifically staying where it is under an lra folder, I wonder if it might be nicer as https://narayana.io/certifications/lra/microprofile-lra-1.0/index.html or https://narayana.io/certifications/microprofile-lra-1.0/index.html but I don't have a particular strong opinion. Given the current state of the PR then @xstefank - it would not cause a problem for the move to take place?

Reply:
personally, I would prefer everything LRA under LRA menu. This will be a nice place to point people after conference talks.

Reply:
@ochaloup do you have any preference?